update contributing.md (#808)
eBay/skin: CONTRIBUTING.md

1. Push commit to origin.
1. Push the git tag to origin, e.g. `git push origin v3.1.0`.
1. Run `npm publish` to publish the package to NPM.

> 1. Publish the `/_cdn/skin/{version}` folder to the CDN using the CDN Upload Portal.

Reviewer: This is only internal to eBay. That's why it's no in the instructions already. E.g. this is an Open source project. Maybe if we want to promote uploading to a CDN we say this?:
```suggestion
1. Publish the `/_cdn/skin/{version}` folder to the CDN of your choice.
```

Author: That's why I left off any internal links.  
But these are instructions for publishing the official package, right?  No one but us should be doing that, so they should be using the CDN Upload Portal.
My main concern is that not many people know how to upload to the CDN.  This at least gives them a clue to search for.  But the other option is that I keep it vague and they ask the team.
What do you think?

Reviewer: I am 100% okay leaving it vague, since the only people doing it for the internal CDN is ... us.

Author: Updated
